include/deal.II/particles/particle.h

```diff
@@ -0,0 +1,306 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__particles_particle_h
+#define dealii__particles_particle_h
+
+#include <deal.II/particles/property_pool.h>
+
+#include <deal.II/base/point.h>
+#include <deal.II/base/types.h>
+#include <deal.II/base/array_view.h>
+
+#include <boost/serialization/vector.hpp>
+
+DEAL_II_NAMESPACE_OPEN
+
+/**
+ * A namespace that contains all classes that are related to the particle
+ * implementation, in particular the fundamental <code>Particle</code> class.
```

Comment:
If I am not mistaken, you can drop `<code>` here and doxygen will link to `Particle` class automatically. But please check.